decentraland/unity-renderer
# Feat: added guest user view of passport

## What does this PR change?

Fix #3331
Adds the guest view structure of the new passport

...

## How to test the changes?

1. Go to: https://play.decentraland.zone/?renderer-branch=feat/guest-passport&ENABLE_new_avatar_flow=
2. Open the passport of a regular user and verify it appears like this:
![Screenshot 2022-11-18 at 10 19 47](https://user-images.githubusercontent.com/4254522/202667354-08b2d5f6-0a66-40b3-b9fd-03d95229485f.png)
3. Open the passport of a guest user and verify it appears like this:
![Screenshot 2022-11-18 at 10 19 39](https://user-images.githubusercontent.com/4254522/202667398-ebe0ea8b-3b0e-4992-8a07-4f6adaf0dacc.png)


## Our Code Review Standards

https://github.com/decentraland/unity-renderer/blob/master/docs/code-review-standards.md
